### PR TITLE
BndTools Tutorial updated

### PR DESCRIPTION
**Pull request by isalvadori**
- update basic tutorial to be consistent with latest bnd version

Signed-off-by: Ilenia Salvadori <i.salvadori@data-in-motion.biz>

### CODE REVIEW

File: tutorial.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Bndtools Tutorial
+itle: Bndtools Tutorial
```
Comment (bjhargrave) [inline tutorial.md]: This seems wrong. You lost the leading `T`.

**Comment by bjhargrave** [comment]
@isalvadori Does this update address the comments in #15? If it does, then make sure to add `Fixes #15` in the commit comment for this PR. Thanks.

File: tutorial.md [hunk 2]
```diff
@@ -1,53 +1,80 @@
 ---
-title: Bndtools Tutorial
+Title: Bndtools Tutorial
```
Comment (bjhargrave) [inline tutorial.md]: I think this should be as it was before `title`.

Comment (bjhargrave) [reply tutorial.md]: @isalvadori Do you plan to update this PR?

Comment (isalvadori) [reply tutorial.md]: @bjhargrave  Yes, sorry for the delay. I will update it today!

**Comment by bjhargrave** [comment]
Thanks so much!

**Pull request merged**